cypress/e2e/variables.cy.js

```diff
@@ -0,0 +1,57 @@
+describe('Variables spec', () => {
+  /* ==== Test Created with Cypress Studio ==== */
+  it('Create a variable', function() {
+    /* ==== Generated with Cypress Studio ==== */
+    // cy.visit('https://stage-console.azion.com');
+    // cy.get('[data-testid="signin-block__email-input"]').clear();
+    // cy.get('[data-testid="signin-block__email-input"]').type('alisson.marcelo+stage2@azion.com{enter}');
+    // cy.get('[data-testid="signin-block__password-input"] > .p-inputtext').clear();
+    // cy.get('[data-testid="signin-block__password-input"] > .p-inputtext').type('LALALALA');
+    // cy.get('[data-testid="signin-block__signin-button"] > .p-button-label').click();
+    // cy.get('.sm\\:p-8 > .flex > .text-color').should('be.visible');
+    /* ==== End Cypress Studio ==== */
+
+    /* ==== Copied from login test ==== */
+    const email = Cypress.env('CYPRESS_EMAIL_STAGE')
+    const password = Cypress.env('CYPRESS_PASSWORD_STAGE')
+    const username = Cypress.env('CYPRESS_USERNAME_STAGE')
```

Comment:
centralizar esse código em uma helper function e reutilizar nos dois testes

Reply:
podemos criar o código como comando custom de login com e-mail e usar o session para evitar criar isso várias vezes nos testes:

adicionar ao commands.js:

function login(email, password) {
  cy.session(
    ['login-with-email-session', email],
    () => {
      cy.visit('/login')
      cy.getByTestId('title').should('have.text', ' Azion Console ')
      cy.getInput('email').type(email)
      cy.getByTestId('next-button').click()
      cy.getInput('password').type(password, { log: false })
      cy.getByTestId('submit').click()
      cy.document().its('cookie').should('contain', 'ajs_user_id')
    },
    {
      cacheAcrossSpecs: true
    }
  )
}

Reply:
Faria em outra issue o reaproveitamento de sessão.